### PR TITLE
fix(plugin-meetings): added alias as option in joinmeeting

### PR DESCRIPTION
**Pull request by Parimala032**
# COMPLETES #[SPARK-552246](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-552246)
## This pull request addresses

This PR adds support for the `alias` parameter to the `joinMeeting` function and updates the corresponding unit tests to verify its correct handling.  

## by making the following changes

**joinMeeting Function:**
Added the text field in a samples page to enter the guest-user-name so the same name will be visible on the remote side as well and this text field name is an optional if user won't enter the name by default the system name will be taken
   - Included the `alias` parameter in the `joinMeeting` function call to ensure it is correctly passed along with other parameters.
These changes are used for the user in such a way that when the user is trying to join the meeting by providing a name according to their wish it's going to be displayed on the remote side. so for this, we are adding one param as `alias` and passing it into the joinMeeting options 

added an text field to enter the guest user name please do watch screen recording

https://github.com/user-attachments/assets/7be17db5-d288-44ba-8f84-9cfc9142e205

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios were tested

Tested this flow by entering the name in a newly added text field, clicking on joinWithMedia button, and checked whether it's working or not by joining in the Meetings sample app it works fine as expected please see the screen recording for reference 
### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/test/unit/spec/meeting/utils.js [hunk 1]
```diff
@@ -138,12 +138,13 @@ describe('plugin-meetings', () => {
         };
 
         MeetingUtil.parseLocusJoin = sinon.stub();
-        await MeetingUtil.joinMeeting(meeting, {});
+        await MeetingUtil.joinMeeting(meeting, {alias: 'userName'});
```
Comment (mkesavan13) [inline packages/@webex/plugin-meetings/test/unit/spec/meeting/utils.js:141]: nitpick: This alias could also be `assigned name` to be uniform and be self-explanatory

**Review by mkesavan13 — CHANGES_REQUESTED** [review]
Just a nitpick comment and rest looks good.

However, can we add a text field in the `Join meeting` section of the sample app to get an alias name? If the box is empty, we need not send anything. This way, it'll be easily testable in future.

**Comment by Parimala032** [comment]
> Just a nitpick comment and rest looks good.
> 
> However, can we add a text field in the `Join meeting` section of the sample app to get an alias name? If the box is empty, we need not send anything. This way, it'll be easily testable in future.

Addressed your comments @mkesavan13

File: packages/@webex/plugin-meetings/test/unit/spec/meeting/utils.js (hunk 1 above)
Comment (Parimala032) [reply packages/@webex/plugin-meetings/test/unit/spec/meeting/utils.js]: @mkesavan13 Updated the code please do have the re-review

**Review by mkesavan13 — APPROVED** [review]

**Review by adhmenon — APPROVED** [review]

**Pull request merged**